src/sage/plot/animate.py

```diff
@@ -436,10 +454,12 @@ def make_image(self, frame, filename, **kwds):
             sage: class MyAnimation(Animation):
             ....:    def make_image(self, frame, filename, **kwds):
             ....:        G = frame.plot()
-            ....:        G.set_axes_range(floor(G.xmin()),ceil(G.xmax()),floor(G.ymin()),ceil(G.ymax()))
+            ....:        G.set_axes_range(floor(G.xmin()), ceil(G.xmax()),
+            ....:                         floor(G.ymin()), ceil(G.ymax()))
             ....:        G.save_image(filename, **kwds)
 
-            sage: B = MyAnimation([graphs.CompleteGraph(n) for n in range(7,11)], figsize=5)
+            sage: B = MyAnimation([graphs.CompleteGraph(n)
+            ....:                  for n in range(7,11)], figsize=5)
             sage: d = B.png()
```

Comment:
I am not sure if we should count leading spaces to check 80 characters limit. 80 characters without counting leading spaces look fine in the rendered doc.

Reply:
OK, I've joined these lines in b5c1e8a996